.travis.yml

```diff
@@ -1,6 +1,3 @@
 language: node_js
 node_js:
-  - "4.0.0"
-  - "5.0.0"
-  - "6.0.0"
   - "stable"
```

Comment:
https://docs.travis-ci.com/user/languages/javascript-with-nodejs/

I still have no idea what version that uses